### DISCUSSION TITLE
appium:forceAppLaunch - is there any way to check the impact of this capability.

### DISCUSSION THREAD

**Discussion by svermagit** [appium/appium#16086] [Q&A] [answered]
Hello.

I have tried this capability in my test and didn't get any impact, could you please suggest any way to see the imapct of this capability on simulator or in appium server logs. Till now I am not getting exactly what and how is impact in my test. Applications are getting launch in same way like wise earlier.

**Comment by KazuCocoa** [comment, marked as answer]
What the capability does is calling `[app launch]` in https://github.com/appium/WebDriverAgent/blob/master/WebDriverAgentLib/Commands/FBSessionCommands.m#L133-L155

The difference is when the value is false, appium does not launch the bundleId process if it was alive.
If you did not provide launching arguments/environments, the impact should be non or very few.
If you provided them, possibly the previous process's ones remain in the next appium session.

I guess you can find something in the system log.
